.browserslistrc

```diff
@@ -1,8 +1,7 @@
-last 2 major versions, 
-> 2%, 
-not dead, 
-not ie > 0, 
-not opera > 0, 
-not op_mini all, 
-not op_mob > 0,
-
+last 2 Chrome version
+last 2 Firefox version
+last 2 Edge major versions
+last 2 Safari major version
+last 2 iOS major versions
+Firefox ESR
```

Comment:
Not required IMHO. If our contracts aren't by default using this, we don't want any adverse effects by supporting it.

Reply:
I do see this seems to be a standard. Will leave as is.